### DISCUSSION TITLE
Armory Crate pop up when pressing M4 key

### DISCUSSION THREAD

**Discussion by llinxx93** [seerge/g-helper#277] [General]
I've bound the M4 key to open the GHelper window however everytime I press it I also get an armory crate pop up which I've already unistalled. Is there a way to stop this from happening?

**Comment by seerge** [comment]
@llinxx93 it's literally first answer in FAQ section
https://github.com/seerge/g-helper#how-do-i-stop-the-armory-crate-install-popup-appearing-every-time-i-press-the-m4--rog-key

**Reply by llinxx93** [reply]
Wow I promise I did read through the main page and discussion posts before asking but apparently I’m illiterate. Thanks for the help I appreciate it
